charts/zabbix/README.md

```diff
@@ -484,11 +488,12 @@ The following tables lists the configurable parameters of the chart and their de
 | zabbixweb.readinessProbe.timeoutSeconds | int | `5` | Number of seconds after which the probe times out |
 | zabbixweb.replicaCount | int | `1` | Number of replicas of ``zabbixweb`` module |
 | zabbixweb.resources | object | `{}` | Requests and limits of pod resources. See: [https://kubernetes.io/docs/concepts/configuration/manage-resources-containers](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers) |
-| zabbixweb.service | object | `{"annotations":{},"clusterIP":null,"port":80,"type":"ClusterIP"}` | Certificate containing certificates for SAML configuration saml_certs_secret_name: zabbix-web-samlcerts |
+| zabbixweb.service | object | `{"annotations":{},"clusterIP":null,"nodePort":31080,"port":80,"type":"NodePort"}` | Certificate containing certificates for SAML configuration saml_certs_secret_name: zabbix-web-samlcerts |
```

Comment:
Why change Web service to NodePort by default? Is this good practice? Personally, I don't want to have NodePort services by default. There are many scenarios where they are either not allowed to use or not desired. Ingress should be the way to go to publish zabbix-web in production, and NodePort or port-forward is for testing scenarios only, imo.

Reply:
Hello @sa-ChristianAnton!

To use ingress, the service must be NodePort. I switched to make it easier. But I understand your concerns and have rolled back.